packages/rancher-backup/package.yaml

```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/backup-restore-operator/releases/download/v1.0.4-rc4/rancher-backup-1.0.4-rc4.tgz
+url: https://github.com/rancher/backup-restore-operator/releases/download/v1.2.0-rc2/rancher-backup-1.2.0-rc2.tgz
```

Comment:
Why is this 1.2.0 instead of 1.0.5 ?

Reply:
@cbron  we decided to use the following versions last time we discussed:
``` 
rancher-backup 1.2.0 in rancher v2.5 
rancher-backup 2.0.0 in rancher v2.6
```